[geoserver] broken startup in stable when not using docker/jetty

this was discussed in #3094 but was never fixed in the doc or in the ansible playbook, so a deployment of 20.0.x or 20.1.x using one of those methods will miserably fail when starting geoserver with CORS enabled:
```
20-Apr-2021 08:06:47.121 GRAVE [Catalina-utility-2] org.apache.catalina.core.StandardContext.filterStart Exception au démarrage du filtre [cross-origin]
        java.lang.ClassNotFoundException: org.eclipse.jetty.servlets.CrossOriginFilter
```

so yeah, somewhere, something has to be done so that 'jetty-servlets' (or whatever that is) are present somewhere ?

fetching jetty-servlets jar into `geoserver/WEB-INF/lib` isnt enough to fix the startup failure:
```
<2>Caused by: org.apache.catalina.LifecycleException: Echec de démarrage du composant [StandardEngine[Catalina].StandardHost[localhost].StandardContext[/geoserver]]
<2>    at org.apache.catalina.util.LifecycleBase.handleSubClassException(LifecycleBase.java:440)
<2>    at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:198)
<2>    at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:717)
<2>    ... 37 more
<2>Caused by: java.lang.NoClassDefFoundError: Lorg/eclipse/jetty/util/log/Logger;
```

--- a/src/web/app/src/main/webapp/WEB-INF/web.xml
+++ b/src/web/app/src/main/webapp/WEB-INF/web.xml
@@ -170,7 +170,7 @@
      -->
    </filter>
 
-   <!-- Uncomment following filter to enable CORS -->
+   <!-- Uncomment following filter to enable CORS in jetty. dont forget to comment the Tomcat CORS filter config below...
    <filter>
         <filter-name>cross-origin</filter-name>
         <filter-class>org.eclipse.jetty.servlets.CrossOriginFilter</filter-class>
@@ -190,6 +190,24 @@
            <param-name>allowedHeaders</param-name>
            <param-value>*</param-value>
        </init-param>
+    </filter>
+   -->
+   <!-- Uncomment following filter to enable CORS in Tomcat. -->
+    <filter>
+      <filter-name>cross-origin</filter-name>
+      <filter-class>org.apache.catalina.filters.CorsFilter</filter-class>
+      <init-param>
+        <param-name>cors.allowed.origins</param-name>
+        <param-value>*</param-value>
+      </init-param>
+      <init-param>
+        <param-name>cors.allowed.methods</param-name>
+        <param-value>GET,POST,PUT,DELETE,HEAD,OPTIONS</param-value>
+      </init-param>
+      <init-param>
+        <param-name>cors.allowed.headers</param-name>
+        <param-value>*</param-value>
+      </init-param>
     </filter>
 
     <!-- 

